commons/data_access_layer/cosmos_db.py

```diff
@@ -133,6 +133,30 @@ def create_sql_condition_for_visibility(
             return 'AND NOT IS_DEFINED(%s.deleted)' % container_name
         return ''
 
+    @staticmethod
+    def create_sql_active_condition(
+        status_value: str, container_name='c'
+    ) -> str:
+        if status_value != None:
+            not_defined_condition = ''
+            condition_operand = ' AND '
+            if status_value == 'active':
+                not_defined_condition = (
+                    'AND NOT IS_DEFINED({container_name}.status)'.format(
+                        container_name=container_name
+                    )
+                )
+                condition_operand = ' OR '
+
+            defined_condition = '(IS_DEFINED({container_name}.status) AND {container_name}.status = \'{status_value}\')'.format(
```

Comment:
Maybe we can split this line for a better readability and to follow the [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) specification.